tests/language_2/abstract_beats_arguments_test.dart

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) 2015, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import "package:expect/expect.dart";
+
+// TODO(rnystrom): This test should be renamed since now it's just about
+// testing that constructing an abstract class generates an error.
+
+abstract class A {
+  A() {}
+}
+
+void main() {
+  /* @compile-error */ new A();
```

Comment:
If you change this to:
```dart
void main() {
  /*@compile-error=unspecified*/ new A();
}
```

Then it will be compatible with the annotation framework I'm already using for type inference tests, and we can replace `unspecified` with the appropriate error code once we know what it is.  (Note: make sure there is no space between `*` and `@`).